p2p_beef_tx.go

```diff
@@ -115,19 +124,17 @@ func decodeBUMPs(beefBytes []byte) ([]BUMP, []byte, error) {
 		bumps = append(bumps, bump)
 	}
 
+	if len(bumps) == 0 {
+		return nil, nil, errors.New("invalid BEEF- lack of BUMPs")
```

Comment:
Maybe another error message? This same is on line 94 so if the error appear  it will be unclear where exactly it came from

Reply:
both should be handled in separated test cases in my opinion

Reply:
I realized this if statement has no sense - there is no option to got bumps collection shorter than declared Bumps without getting errors. I remove this